chainer/functions/connection/convolution_nd.py

```diff
@@ -349,21 +384,56 @@ def convolution_nd(x, W, b=None, stride=1, pad=0, use_cudnn=True,
             `cover_all` needs to be ``False`` if you want to use cuDNN.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable:
+            Output variable of shape :math:`(n, c_O, l_1, l_2, ..., l_N)`.
 
-    This function uses cuDNN implementation for its forward and backward
-    computation if ALL of the following conditions are satisfied:
+    .. note::
 
-    - ``cuda.cudnn_enabled`` is ``True``
-    - ``use_cudnn`` is ``True``
-    - The number of spatial dimensions is more than one.
-    - ``cover_all`` is ``False``
-    - The input's ``dtype`` is equal to the filter weight's.
-    - The ``dtype`` is FP32, FP64 or FP16(cuDNN version is equal to or greater
-      than v3)
+        This function uses cuDNN implementation for its forward and backward
+        computation if ALL of the following conditions are satisfied:
+
+        - ``cuda.cudnn_enabled`` is ``True``
+        - ``use_cudnn`` is ``True``
+        - The number of spatial dimensions is more than one.
+        - ``cover_all`` is ``False``
+        - The input's ``dtype`` is equal to the filter weight's.
+        - The ``dtype`` is FP32, FP64 or FP16(cuDNN version is equal to or
+          greater than v3)
```

Comment:
Thank you. The current description could be misleading because user could think that the condition cuDNN >= v3 must be hold even FP32 or FP64 cases.